lib/ClimaLandSimulations/src/utilities/met_drivers_FLUXNET.jl

```diff
@@ -0,0 +1,208 @@
+function setup_drivers(site_ID)
+
+    af = ArtifactFile(
+        url = data_link,
+        filename = "AMF_$(site_ID)_FLUXNET_FULLSET.csv",
+    )
+    dataset = ArtifactWrapper(
+        "$(@__DIR__)" * "/$site_ID",
+        "ameriflux_data",
+        ArtifactFile[af],
+    );
+    dataset_path = get_data_folder(dataset);
```

Comment:
@Espeer5 I noticed that this line of code wants the fluxnet site folders to be located in the same folder as this script (met_drivers_FLUXNET.jl) 

How would you change it to `lib/ClimaLandSimulations/src/fluxnet`?

Reply:
Possibly `joinpath(ClimaLSM_dir, "/lib/ClimaLandSimulations/src/fluxnet")` where ClimaLSM_dir is the package dir of CliMALSM?

Alternatively if ClimaLandSimulations is a package perhaps we could now get the package dir of that itself and `joinpath` with just `/src/fluxnet`

Reply:
I tried but I think it's not the right way to go. Maybe I need to change `dataset` instead

Reply:
Ah yes the dataset path when saved on line 10 `$(@__DIR__)" * "/$site_ID` is the issue. You need the dataset to be something like 

``` 
dataset = ArtifactWrapper(
        "$package_dir(ClimaLandSimulations)" * "/$site_ID",
        "ameriflux_data",
        ArtifactFile[af],
    )
  ```
   And then the rest of the code should remain as is. I am trying to test out but waiting for precompilation

Reply:
Okay I just tested this out and I was successful with:
 - Removing all of the site-ID directories within the utilities folder
 - Editing the `dataset` call to:
 ```
dataset = ArtifactWrapper(
        "$ClimaLandSimulations_dir/src/fluxnet/$site_ID",
        "ameriflux_data",
        ArtifactFile[af],
    )
```
And not changing anything else.

Reply:
Thanks Ed!! I'll add that asap